### DISCUSSION TITLE
Stenography

### DISCUSSION THREAD

**Discussion by Symbreaker** [monkeytypegame/monkeytype#4471] [Ideas]
Stenography or stenotype is the way court reporters are able to keep up with people speaking in court. Having an in house emulator would help people get over the barrier to entry and more enthusiasts to know about steno. Steno uses chords (multiple keys pressed together), not all keyboards have n-key rollover so Plover allows you to hold onto a key and only output when you release all keys. 

I hope this isn't asking for much:
- emulate standard steno
- display the chording for each word on a keymap below the text
It could be expanded to include more theories and custom theory dictionaries later

I hope you can just source from:
https://github.com/openstenoproject

This would be a competitor website:
https://didoesdigital.com/typey-type/

This wiki could help with development if you need to know how something works:
https://plover.readthedocs.io/en/latest/system_dev.html

Plover is the software people install to be able to use steno on their computer and it contains Spectra Lexer which helps you look up word chordings. People can use steno on monkeytype now by first globally doing it on your computer and add a tag to not overwrite your score.

**Comment by Miodec** [comment]
IMO recreating a whole steno system sounds like a bit of a pointless effort, espeicially since installing Plover takes couple minutes.

**Reply by Symbreaker** [reply]
Yes, and I also know it is possible to add a tag too, so your stenotyping doesn't overwrite your normal typing speeds. You could get other alt layouts to work globally on your computer in the same way, but I like that alt layouts are in monkeytype. On the worry of just recreating, I should have mentioned that they fall under the open steno project and hopefully can be just copied in many cases. But if there are any problems that could arise from such a thing, such as licensing or lots of research and linking pls let me know.

**Reply by Miodec** [reply]
Yes, alt layouts are on monkeytype because emulating a traditional input method is very very very easy. Emulating steno is not easy at all. There are many areas for possible issues and bugs, and I dont think its worth it for me to maintain this code, especially when Plover is so easily available.
